test/srt/test_update_weights_from_tensor.py

```diff
@@ -1,32 +1,40 @@
+import time
 import unittest
 
 import torch
 
 import sglang as sgl
-from sglang.test.test_utils import DEFAULT_SMALL_MODEL_NAME_FOR_TEST
+from sglang.test.test_utils import DEFAULT_MODEL_NAME_FOR_TEST
```

Comment:
can we use the small model for faster test?

Reply:
Sure, I originally used the bigger one to test speedup and forget to change back

Reply:
Done (waiting for CI)